### ISSUE TITLE
[important] Free filter should also show Free with certificate

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-course-list#336]
Use case: Individual wants a free course. They select the filter "Free of charge" (and don't bother noticing the next filter).

Currently, the courses that are "Free, with certificates for purchase" are filtered out, and this user doesn't see them at all.

I think this is a design error.

Ideas for addressing this:
1. Maybe short-term fix: Change the filter code so that if user selects "Free of charge", the list also displays "Free, with certificates for purchase"
2. In the submission form, change "cost" from single-select option button to multiple-select checkbox. This might be needed for other situations in the future. For example, I expect there are some courses that have **both** "Paid, with some free content or free for a limited duration" and "Paid, with reduced fee for some". I think this could wait until next iteration if needed.

W3C-selfishly, I'd like this addressed (at least 1. or something like it) before we announce the course list. (Although, since we have the W3C course at the top, I can accept announcing before it's fixed, as long as it's soon.

**Comment by leticiaseixas** [comment]
Hi @shawna-slh,

I think the use case mentioned can be solved just by changing the filters: combining the two filters ("Free of charge", and "Free, with certificates for purchase") and displaying in more detail in the list, similar to what was done with Type of resource. So we can make sure that this change would be made soon, while we discuss further the other use case mentioned regarding paid courses.

**Comment by shawna-slh** [comment]
+1 

I think combining those 2 filters falls under the EOWG [Process for Minor Changes](https://www.w3.org/WAI/EO/wiki/EOWG_Participation_Info#Process_for_Minor_Changes). @kakinney and @bakkenb can weigh in.

That means you can go ahead and make the change, and we just need to notify EOWG. We can add to Work for this Week or elsewhere. In W4tW, there's a commented out section:
```
<!--
===Minor Changes for Review===
The updates below were made under the [https://www.w3.org/WAI/EO/wiki/EOWG_Participation_Info#Process_for_Minor_Changes EOWG Process for Minor Changes]. If you have any concerns with or comments on these changes, please submit a GitHub issue (or e-mail wai-eo-editors@w3.org)
* ...
-->
```

**Comment by leticiaseixas** [comment]
Combined "Free of charge" and "Free, with certificates for purchase: [Preview](https://deploy-preview-339--wai-course-list.netlify.app/courses/list/)

**Comment by bakkenb** [comment]
+1  - minor change

**Comment by leticiaseixas** [comment]
resolved with #339
